gdrive

```diff
@@ -57,16 +57,19 @@ def init():
     return service
 
 def get_filename(service, file_id):
-    """Gets the file name for a file ID"""
-    filename = 'gdrive-file'
+    """Gets the file name for a file ID and describes its info"""
+    filedata = None
     try:
-        file_metadata = service.files().get(fileId=file_id, fields='name').execute()
-        filename = file_metadata['name']
-    except Exception as err:
-        print('An error happened while getting file name.')
-        print(err)
+        fields = 'id, name, size, modifiedTime, modifiedByMeTime, owners'
+        file_metadata = service.files().get(fileId=file_id, fields=fields).execute()
```

Comment:
I think this could be done in a separate method. A 'get_file_metadata' method so we don't have all of this other fields defined in the 'get_filename' method.

Reply:
Changed get_filename to get_metadata and removed the "name" operations